### DISCUSSION TITLE
An estimation of when pigment will be ready for production

### DISCUSSION THREAD

**Discussion by yishay-at-bay** [mui/pigment-css#356] [Discussions]
Hi

I'm using MUI v6 with styled components
As I understood, the old styling system (emotion/styled) does not really support Server Components and this is part of the reason to migrate to pigment
Can someone please give some estimation and tell when pigment will be ready for use in production?

**Comment by tobbas** [comment]
Any chance we could get some feedback on this?

**Comment by MikaelColliander** [comment]
I'm also curious. Is the project still alive?

**Reply by EnzoAlbornoz** [reply]
Definitely dead, 1 year without one single update. They couldn't even bump their dependency versions, which would have helped a lot. Each day, MUI turns out to be more of a burden than a tool.
